### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/fbopen#195]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)

## Results for https://fbopen.gsa.gov

* __Error__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * `#github-link`
 * `<a id="github-link" href="https://github.com/18f/fbopen"></a>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.44:1. Recommendation: change text colour to #00798f.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#intro > p:nth-child(3) > small > a:nth-child(2)`
 * `<a href="https://pages.18f.gov/fbopen/">API documentation</a>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.54:1. Recommendation: change text colour to #6e6e6e.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#footer > div > p:nth-child(1)`
 * `<p class="landing-copy-small">
        FBOpen and this sample...</p>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.44:1. Recommendation: change text colour to #00798f.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#footer > div > p:nth-child(1) > a:nth-child(2)`
 * `<a href="https://18f.gsa.gov">18F</a>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.48:1. Recommendation: change text colour to #d22e07.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#intro > p:nth-child(3) > small`
 * `<small>Developers, learn more at our <...</small>`

* __Error__ This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
 * `#q-search`
 * `<button type="submit" id="q-search"><span class="glyphicon glyphico...</button>`

* __Error__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * `#email-link`
 * `<a id="email-link" href="mailto:fbopen@gsa.gov"></a>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.03:1. Recommendation: change text colour to #009aae.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
 * `#intro > div > span`
 * `<span>pilot</span>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.48:1. Recommendation: change text colour to #d22e07.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#intro > p:nth-child(2)`
 * `<p class="landing-copy">
        FBOpen helps small bus...</p>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.44:1. Recommendation: change text colour to #00798f.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#intro > p:nth-child(3) > small > a:nth-child(1)`
 * `<a href="https://github.com/18f/fbopen">github repo</a>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.44:1. Recommendation: change text colour to #00798f.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `#footer > div > p:nth-child(1) > a:nth-child(3)`
 * `<a href="http://whitehouse.gov/innovationfellows">Presidential Innovation Fellows</a>`

* __Error__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * `#twitter-link`
 * `<a id="twitter-link" href="https://twitter.com/fbopen"></a>`

### Summary

__Errors:__ 12
